README.md

```diff
@@ -14,7 +14,7 @@ The CloudEvents SDK for JavaScript.
 - Serialize and deserialize CloudEvents in different [event formats](https://github.com/cloudevents/spec/blob/v1.0/spec.md#event-format).
 - Send and recieve CloudEvents with via different [protocol bindings](https://github.com/cloudevents/spec/blob/v1.0/spec.md#protocol-binding).
 
-_Note:_ Supports CloudEvent versions 0.3, 1.0
```

Comment:
no 0.3? Doesn't the spec say we should support 0.3?

This change seems tangential to the rest of the change.

Reply:
0.3 support was removed last August in https://github.com/cloudevents/sdk-javascript/pull/425 but this line was missed in the removal. Support for 0.3 is considered optional, so it was removed in favor of simplicity. There may still be some unfortunate remnants around, like this, that need clean up.